docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  gqlgen-server:
+    build:
+      context: examples/gqlgen-service
+    healthcheck:
```

Comment:
As far as I know healthchecks don't influence `docker-compose` startup order.
https://docs.docker.com/compose/startup-order/#title see the _tip_ infobox

I see below you've used the `service_healthy` condition, I can find examples of that but there is nothing in the docker-compose yaml documentation for it.

Reply:
They don't by default but that means the bramble container dies if you have to build any of the dependent images. 

From what I understand, it was a feature in v2, they removed it in v3 because it caused issues with swarm. They then merged v2 and v3 config formats but kept it undocumented, probably because of swarm again. 

Given it's useful and we have no interest in swarm I'd vote to use it as long as it works.